torch/_inductor/kernel/flex_decoding.py

```diff
@@ -297,6 +297,8 @@ def flex_decoding_grid(batch_size, kv_heads, gqa_group_size, n_keys, d_model, me
 def get_split_k(B: int, H: int, Mk: int, SM: int = 128) -> int:
     """Heuristic for the number of splits from xformer"""
     bh = max(B * H, 1)  # NOTE: Handle B*h=0 case
+    # Defaults to two splits if symbolic
+    bh = V.graph.sizevars.size_hint(bh, fallback=2)
```

Comment:
Spoke offline and we think we should just specialize on shape and only do flex_decoding if bh is not dynamic

Reply:
So you are going to change this code?